okhttp-testing-support/src/main/java/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -88,15 +89,18 @@ class OkHttpClientTestRule : TestRule {
     testClient?.let {
       val connectionPool = it.connectionPool
       connectionPool.evictAll()
-      assertThat(connectionPool.connectionCount()).isEqualTo(0)
+      assertEquals(0, connectionPool.connectionCount())
     }
   }
 
   private fun ensureAllTaskQueuesIdle() {
-    for (queue in TaskRunner.INSTANCE.activeQueues()) {
-      assertThat(queue.idleLatch().await(1_000L, TimeUnit.MILLISECONDS))
-          .withFailMessage("Queue still active after 1000 ms")
-          .isTrue()
+    try {
+      for (queue in TaskRunner.INSTANCE.activeQueues()) {
+        assertTrue("Queue still active after 1000 ms",
+            queue.idleLatch().await(1_000L, TimeUnit.MILLISECONDS))
+      }
+    } finally {
+      TaskRunner.INSTANCE.cancelAll()
```

Comment:
should this be an if/else thing?

```
for (queue in TaskRunner.INSTANCE.activeQueues()) {
  if (!queue.idleLatch().await(1_000L, TimeUnit.MILLISECONDS)) {
    TaskRunner.INSTANCE.cancelAll() 
    fail("Queue still active after 1000 ms")
  }
}
```
that way we don’t cancelAll() when it’s succeeding?